linera-core/src/remote_node.rs

```diff
@@ -191,31 +191,23 @@ impl<N: ValidatorNode> RemoteNode<N> {
     pub(crate) async fn find_missing_blobs(
         &self,
         blob_ids: Vec<BlobId>,
-        chain_id: ChainId,
+        locked_published_blobs: &[Blob],
+        locked_used_blobs: &[Blob],
```

Comment:
The method documentation sounds like we'd download the validator's chain info. Instead, the assumption is that we already did that, and that we're passing the blobs form that chain info in here.